app/views/account/_auth_providers.html.erb

```diff
@@ -39,12 +39,13 @@ See doc/COPYRIGHT.rdoc for more details.
 auth_provider_html = call_hook :view_account_login_auth_provider
 no_pwd = OpenProject::Configuration.disable_password_login?
 pclass = no_pwd ? 'no-pwd' : ''
+wclass = local_assigns[:wide] ? 'wide' : ''
 %>
 <% if auth_provider_html.strip != '' %>
-  <div class="login-auth-providers <%= pclass %>">
+  <div class="login-auth-providers <%= pclass %> <%= wclass %>">
     <% unless no_pwd %>
       <h3 class="login-auth-providers-title"><span>
-          <%= I18n.t('account.login_with_auth_provider')%>
+          <%= local_assigns[:omniauth_title] || I18n.t('account.login_with_auth_provider')%>
```

Comment:
Always thought `local_assigns` was considered a hack outside rails internals, but that doesn't seem to be the case anymore, since i has received documentation in the meantime. (http://nithinbekal.com/posts/rails-optional-locals/)